### PR TITLE
fix lint errors

### PR DESCRIPTION
**Pull request by dejii**
- fix lint errors generated by https://github.com/re-data/re-data/pull/227

### CODE REVIEW

File: re_data_ui/src/utils/helpers.ts [hunk 1]
```diff
@@ -104,6 +104,9 @@ export const appendToMapKey = (
 export const supportedResTypes = new Set(['source', 'model', 'seed']);
 
 export const generateModelId = (details: DbtNode | DbtSource): string => {
-  const { database, schema, name, identifier } = details;
-  return `${database}.${schema}.${identifier || name}`.toLowerCase();
+  const {
+    database, schema,
+  } = details;
+  const identifier = details.resource_type === 'source' ? (details as DbtSource).identifier : details.name;
```
Comment (mateuszklimek) [inline re_data_ui/src/utils/helpers.ts:110]: @dejii isn't it a different logic?
Meaning for source we also maybe using `name` we just try `identifier` first
?

Comment (dejii) [reply re_data_ui/src/utils/helpers.ts:110]: Right, working with a theory that `identifier` is usually present in `sources`. I will make a change not to assume that.

File: re_data_ui/src/utils/helpers.ts [hunk 2]
```diff
@@ -107,6 +107,12 @@ export const generateModelId = (details: DbtNode | DbtSource): string => {
   const {
     database, schema,
   } = details;
-  const identifier = details.resource_type === 'source' ? (details as DbtSource).identifier : details.name;
+  let identifier;
+  if (details.resource_type === 'source') {
+    const detail = details as DbtSource;
```
Comment (mateuszklimek) [inline re_data_ui/src/utils/helpers.ts:112]: I think naming it detail (just removing s) may be a bit confusing 😅

Comment (mateuszklimek) [reply re_data_ui/src/utils/helpers.ts:112]: a bit of nit picking

Comment (dejii) [reply re_data_ui/src/utils/helpers.ts:112]: I was trying to avoid [eslint/no-param-reassign](https://eslint.org/docs/rules/no-param-reassign) 😅

Comment (dejii) [reply re_data_ui/src/utils/helpers.ts]: that works, thanks!

**Comment by z3z1ma** [comment]
Whoops, did some double work there in #229 

:nerd_face:

**Comment by dejii** [comment]
> Whoops, did some double work there in #229
> 
> 🤓

Ahh thanks @z3z1ma, currently tests run on branches only but we’d look at running some of them on pull requests to catch such issues earlier.

**Review by akinolu52 — APPROVED** [review]

**Pull request merged**